tree/ntuple/v7/test/ntuple_show.cxx

```diff
@@ -374,26 +383,27 @@ TEST(RNTupleShow, Collections)
    std::ostringstream osData;
    reader->Show(0, osData);
    // clang-format off
-   std::string outputData{ std::string("")
-      + "{\n"
-      + "  \"myCollection\": [{\"myShort\": 1, \"myFloat\": 10}, {\"myShort\": 2, \"myFloat\": 20}]\n"
-      + "}\n" };
+   std::string outputData{
+R"({
+  "myCollection": [{"myShort": 1, "myFloat": 10}, {"myShort": 2, "myFloat": 20}]
+}
+)" };
    // clang-format on
    EXPECT_EQ(outputData, osData.str());
 
    std::ostringstream osFields;
    reader->PrintInfo(ROOT::Experimental::ENTupleInfo::kSummary, osFields);
    // clang-format off
-   std::string outputFields{ std::string("")
-      + "************************************ NTUPLE ************************************\n"
-      + "* N-Tuple : Collections                                                        *\n"
-      + "* Entries : 1                                                                  *\n"
-      + "********************************************************************************\n"
-      + "* Field 1           : myCollection                                             *\n"
-      + "*   Field 1.1       : _0                                                       *\n"
-      + "*     Field 1.1.1   : myShort (std::int16_t)                                   *\n"
-      + "*     Field 1.1.2   : myFloat (float)                                          *\n"
-      + "********************************************************************************\n" };
+   std::string outputFields{
+      "************************************ NTUPLE ************************************\n"
```

Comment:
This portion is better matched the indentation/format.

Reply:
Not sure if I interpret this comment correctly: you mean you like this more than the raw strings? In this case I could do without a raw string because there are no tons of `"` to escape, so it works; in all other cases I personally find the raw string much more readable.
However, as it's a subjective choice, the only way to decide is by majority vote I think.